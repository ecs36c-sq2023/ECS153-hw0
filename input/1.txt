Human
5
abcdefg
e e
aaa
Peter
Ada
