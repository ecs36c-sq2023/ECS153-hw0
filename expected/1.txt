e
abcdefg
aaa
Peter
Ada
